roles/kuryr_node/tasks/main.yaml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Create CNI bin directory
+  file:
+    state: directory
+    path: "{{ cni_bin_dir }}"
+    mode: 0755
+    owner: root
+    group: root
+    recurse: yes
+
+- name: Download and extract standard CNI plugins
+  become: yes
+  unarchive:
+    remote_src: True
+    src: "{{ cni_bin_url }}"
+    dest: "{{ cni_bin_dir }}"
+
+- name: Ensure CNI net.d exists
+  file:
+    path: /etc/cni/net.d
+    recurse: yes
+    state: directory
+
+- name: Check for node config
+  stat: "path=/etc/sysconfig/{{ openshift.common.service_type }}-node"
+  register: node_config_check
+
+- name: Configure OpenShift node with disabled service proxy
```

Comment:
This belongs in the node template.

Reply:
What do you mean? Where should I put the task?

Reply:
Never mind, thought this was a different file.